hedera-node/hedera-network-admin-service-impl/src/test/java/com/hedera/node/app/service/networkadmin/impl/test/handlers/FreezeHandlerTest.java

```diff
@@ -81,6 +81,8 @@ class FreezeHandlerTest {
     @Mock(strictness = LENIENT)
     private Account account;
 
+    private final FileID fileUpgradeFileId = FileID.newBuilder().fileNum(150L).build();
+
```

Comment:
It would be good to have a test for other file Ids too, both valid and invalid range

Reply:
sure I will add some tests

Reply:
Looks good